pkg/yurthub/metrics/metrics.go

```diff
@@ -44,15 +44,16 @@ var (
 )
 
 type HubMetrics struct {
-	serversHealthyCollector              *prometheus.GaugeVec
-	inFlightRequestsCollector            *prometheus.GaugeVec
-	inFlightRequestsGauge                prometheus.Gauge
-	inFlightMultiplexerRequestsCollector *prometheus.GaugeVec
-	inFlightMultiplexerRequestsGauge     prometheus.Gauge
-	closableConnsCollector               *prometheus.GaugeVec
-	proxyTrafficCollector                *prometheus.CounterVec
-	errorKeysPersistencyStatusCollector  prometheus.Gauge
-	errorKeysCountCollector              prometheus.Gauge
+	serversHealthyCollector               *prometheus.GaugeVec
+	inFlightRequestsCollector             *prometheus.GaugeVec
+	inFlightRequestsGauge                 prometheus.Gauge
+	aggregatedInFlightRequestsCollector   *prometheus.GaugeVec
+	aggregatedInFlightRequestsGauge       prometheus.Gauge
+	targetForMultiplexerRequestsCollector *prometheus.GaugeVec
```

Comment:
Multiplexer->Aggregated ？

Reply:
I will rename multiplexer to lite-apiserver in a new pull request.